homeassistant/components/sensor/fail2ban.py

```diff
@@ -38,8 +41,9 @@
 })
 
 
-async def async_setup_platform(hass, config, async_add_entities,
-                               discovery_info=None):
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_entities,
+                         discovery_info=None):
```

Comment:
Revert this change. We use the new syntax now.

Reply:
@fabaff do you mean using:
`
async def async_setup_platform(hass, config, async_add_entities,
                                       discovery_info=None):
`